5calls/app/src/main/res/values/strings.xml

```diff
@@ -477,9 +477,13 @@
     <!-- Filter for showing all issues [CHAR_LIMIT=50] -->
     <string name="all_issues_filter">All issues</string>
 
+    <!-- Filter for showing starred issue [CHAR_LIMIT=50] -->
+    <string name="starred_issues_filter">Starred issues</string>
+
     <string-array name="default_filters">
         <item>@string/top_issues_filter</item>
         <item>@string/all_issues_filter</item>
+        <item>@string/starred_issues_filter</item>
```

Comment:
Can we hide and/or disable this unless the user has at least 1 starred issue in the current list?